shaman/controllers/repos/projects.py

```diff
@@ -16,6 +16,12 @@ def __init__(self, project_name):
     def index(self):
         abort(405)
 
+    @index.when(method='GET', template='json')
+    def index_get(self):
+        if not self.project:
+            abort(404)
+        return self.project
```

Comment:
Goo question. If we do it that way we wouldn't have an endpoint where we could consume the actual repo information though. Unless I add a field or separate both. Something like:

    dict(
        "project": self.project,
        "repos": self.project.repos
     )

Or maybe try overload the json coming from the Project model?